<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="appResource.qrc">
    <normaloff>:/Resources/SpeechCalTi.png</normaloff>:/Resources/SpeechCalTi.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.958000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,stop: 0 rgb(72, 85, 99), stop: 1 rgb(41, 50, 60));
color:white;</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="listenTimeCBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    spacing: 5px;
	background-color:rgb(0,0,0,0%);
}

QCheckBox::indicator {
    width: 50px;
    height: 50px;
}

QCheckBox::indicator:unchecked {
    image: url(:/Resources/unchecked.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/Resources/unchecked_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/Resources/unchecked_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/Resources/checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/Resources/checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/Resources/checked_pressed.png);
}

</string>
   </property>
   <property name="text">
    <string>Phrase Time Limit</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="BanglaResCBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    spacing: 5px;
	background-color:rgb(0,0,0,0%);
}

QCheckBox::indicator {
    width: 50px;
    height: 50px;
}

QCheckBox::indicator:unchecked {
    image: url(:/Resources/unchecked.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/Resources/unchecked_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/Resources/unchecked_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/Resources/checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/Resources/checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/Resources/checked_pressed.png);
}

</string>
   </property>
   <property name="text">
    <string>Result In Bangla</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QSlider" name="listenTimeSlider">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider{
	background-color: rgb(0,0,0,0%);
}
QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 15px;
border-radius: 6px;
}

QSlider::sub-page:horizontal {
background-color:qlineargradient(spread:reflect, x1:1, y1:0, x2:0.995, y2:1, stop:0 rgba(218, 218, 218, 255), stop:0.305419 rgba(235, 0, 0, 255), stop:0.935961 rgba(227, 64, 64, 255), stop:1 rgba(240, 240, 240, 255));
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #DCE35B, stop:1 #45B649);
border: 1px solid #777;
width: 13px;
height:30px;
margin-top:-8px;
margin-bottom:-8px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0,0%);
color:white;</string>
   </property>
   <property name="text">
    <string>Set Time</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="appResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
